src/domains/profile/components/SearchAddress/SearchAddress.tsx

```diff
@@ -84,12 +85,12 @@ export const SearchAddress = ({
 			onSearch={onSearch}
 		>
 			<Table columns={columns} data={wallets}>
-				{({ address, walletName, balance, fiat }: any, index: number) => (
+				{(wallet: Wallet, index: number) => (
 					<SearchAddressListItem
-						address={address}
-						walletName={walletName}
-						balance={balance}
-						fiat={fiat}
+						address={wallet.address()}
+						walletName={wallet.alias()}
+						balance={wallet.balance().toString()}
+						fiat={wallet.convertedBalance().toString()}
```

Comment:
```suggestion
						fiat={wallet.convertedBalance()..toHuman(8)}
```

Reply:
This should also be renamed to `convertedBalance` because it can be crypto or fiat.

Reply:
Do you mean `convertedBalance={wallet.convertedBalance().toHuman(8)}}`?

Reply:
Yeah, typo.